tests/foreman/api/test_ping.py

```diff
@@ -21,6 +21,7 @@
 pytestmark = [pytest.mark.tier1, pytest.mark.upgrade]
 
 
+@pytest.mark.e2e
 @pytest.mark.build_sanity
 def test_positive_ping(target_sat):
     """Check if all services are running
```

Comment:
cli/test_ping can also be considered for e2e, right?

Reply:
Having only tested with one interface should be fine , thats where we settled on right @JacobCallahan ?

Reply:
Well, probably not strictly e2e, though it felt like a basic gating test that could run against the stream